### DISCUSSION TITLE
Bundle as a single web component

### DISCUSSION THREAD

**Discussion by aszenz** [jupyterlite/jupyterlite#1368] [Ideas]
To make it easy to embed in web-apps, create a web component which dynamically imports the required sources and kernels based on props.

Something like this:

```html
<jupyter-lite kernels="pyodide" extensions="" themes="">
</juypter-lite>
```
This will also require exporting pkgs to a js cdn like https://www.jsdelivr.com/

**Comment by bollwyvl** [comment]
Good luck with that!
